internal/ui/list_layout.go

```diff
@@ -106,7 +106,7 @@ func (s *ListLayout) appendSection(title, note string, sectionBody []string) {
 		if s.style.NoteSeparator {
 			s.appendLine()
 		}
-		s.l.AppendItem(DefaultNoteColours.Sprintf(note))
+		s.l.AppendItem(DefaultNoteColours.Sprintf("%s", note))
```

Comment:
Is there `Sprint` method available? If yes, the extra _string as string_ format could be omitted 🤔

```suggestion
		s.l.AppendItem(DefaultNoteColours.Sprint(note))
```